src/hash/component.rs

```diff
@@ -89,13 +90,13 @@ impl GridHash {
     /// Intentionally left private, so we can ensure the only place these are constructed/mutated is
     /// this module. This allows us to optimize change detection using [`ChangedGridHashes`].
     #[inline]
-    pub(super) fn new(parent: &Parent, cell: &GridCell) -> Self {
-        Self::from_parent(parent.get(), cell)
+    pub(super) fn new(parent: &ChildOf, cell: &GridCell) -> Self {
+        Self::from_parent(parent.parent, cell)
     }
 
     #[inline]
     pub(super) fn from_parent(parent: Entity, cell: &GridCell) -> Self {
-        let hasher = &mut AHasher::default();
+        let hasher = &mut FixedState::with_seed(0).build_hasher();
```

Comment:
Unsure of correctness, I opted to assume that this expects a consistent hashing state every time without further verification aside from "well it passes tests".